database/Models/index.js

```diff
@@ -7,22 +7,19 @@ const sequelize = new Sequelize({
   database: 'sdc',
   password: 'test',
```

Comment:
You suggested to me to put your password in your env file. I think you were going to do that at some point.

Reply:
yes will get this done next PR!